src/api/modules/utils.ts

```diff
@@ -2,15 +2,13 @@ import jwt from 'jsonwebtoken'
 import axios from 'axios'
 import chalk from 'chalk'
 import { execSync } from 'child-process-es6-promise'
-import Table from 'cli-table2'
```

Comment:
Another improvement: remove `cli-table2` as a dependency. Currently is even generating a security alert because it depends on an old version of loadash.

Reply:
Removed in 053779b6273e0f0a312f4b967c44be44ebfe4f9b, thanks!